docs/source/app_developers_guide/aws.rst

```diff
@@ -1,34 +1,37 @@
-*********************
-Using Sawtooth on AWS
-*********************
+******************************************
+Using AWS for Your Development Environment
+******************************************
+
+This tutorial describes how to set up Hyperledger Sawtooth for application
+development using the Amazon Web Services (AWS) cloud on the Amazon Elastic
+Compute Cloud (Amazon EC2). This step-by-step procedure shows you how to
```

Comment:
I would change "using the Amazon Web Services (AWS) cloud on the Amazon Elastic Compute Cloud (Amazon EC2)" to "using the Amazon Elastic Compute Cloud (Amazon EC2) service"